src/redux/categories.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+const categoriesSlice = createSlice({
+  name: 'categories',
```

Comment:
- [ ] Kindly check the comments above :arrow_up: . I think as developers, we shall always be required to follow strictly a client requirement. We really need to understand how to make our app scalable in the future using ducks pattern.